Specs/Number/English/NumberModel.json

```diff
@@ -3011,5 +3011,21 @@
         "End": 17
       }
     ]
+  },
+  {
+    "Input": "forty two and a next",
```

Comment:
Can you add an example that's a real sentence? This is very valuable for proper evaluation and comparison between models/systems.
Can you also add other cases that may be affected negatively by your change related to "a"?